index.js

```diff
@@ -1,44 +1,58 @@
-const mongoose = require('mongoose');
 
-// Import of the model Recipe from './models/Recipe.model.js'
+const mongoose = require('mongoose');
 const Recipe = require('./models/Recipe.model');
-// Import of the data from './data.json'
-const data = require('./data');
-
-const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
-
-//Method 1 : Using Async Await
+const recipesData = require('./data.json'); // Adjust the path as needed
 
-const manageRecipes = async () => {
+async function main() {
   try {
-    // Connection to the database "recipe-app"
-    const dbConnection = await mongoose.connect(MONGODB_URI);
-    console.log(`Connected to the database: "${dbConnection.connection.name}"`);
-
-    // Before adding any recipes to the database, let's remove all existing ones
-    await Recipe.deleteMany();
-
-    // Run your code here, after you have insured that the connection was made
+    // Connect to the database
+    await mongoose.connect('mongodb://127.0.0.1:27017/receipe-app', {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    });
+
+    console.log('Connected to the database');
+
+    // Iteration 2: Create a recipe
+    const newRecipe = {
+      title: 'Delicious Pasta Carbonara',
+      level: 'Amateur Chef',
+      ingredients: ['200g spaghetti', '100g pancetta', '2 large eggs', '50g Pecorino cheese', 'Salt and black pepper'],
+      cuisine: 'Italian',
+      dishType: 'main_course',
+      image: 'https://example.com/pasta-carbonara.jpg',
+      duration: 30,
+      creator: 'John Doe',
+      created: new Date(),
```

Comment:
Since you have on your model, the default value for the date. You don't need to pass a `created` value, because it will be created with the `Date.now` method